torch/csrc/jit/python/script_init.cpp

```diff
@@ -778,6 +781,18 @@ void initJitScriptBindings(PyObject* module) {
   // NOLINTNEXTLINE(bugprone-unused-raii)
   py::class_<c10::Capsule>(m, "Capsule");
 
+  py::class_<MobileCode>(m, "MobileCode")
+      .def(py::init<std::shared_ptr<Graph>, std::string>())
+      .def("instructions", [](MobileCode& self) -> std::vector<IValue> {
```

Comment:
Is it just for instructions? As a complete frame would it also include other tables like constants, operators, types, etc?

Reply:
Yep I wasn't sure whats needed from mobile side. Those things can be included easily